webapp/channels/src/components/backstage/components/backstage_list.tsx

```diff
@@ -95,7 +95,7 @@ const BackstageList = ({searchPlaceholder = localizeMessage({id: 'backstage_list
                         id='emptySearchResultsMessage'
                         key='emptyTextSearch'
                     >
-                        {React.cloneElement(remainingProps.emptyTextSearch, {values: {searchTerm: filterLowered}})}
+                        {React.cloneElement(remainingProps.emptyTextSearch, {values: {...remainingProps.emptyTextSearch.props.values, searchTerm: filterLowered}})}
```

Comment:
why did we change this?

Reply:
@M-ZubairAhmed 
Kept getting this error:
"The intl string context variable "p" was not provided to the string..." from react-intl.
Figured out that, if we don't pass the rest of the values, in this case the <b> tag, then it becomes undefined causing the error. Tried other ways but this is the only one that seemed to work. I'm open to any ideas if you have a better way.

Reply:
oh i see, can you also cross check and post the screen shot that the integration bots page where it is used is correctly rendering it?

Reply:
Will do.